internal/k8s/ignore.go

```diff
@@ -31,8 +28,40 @@ func (i *IgnoreWrapper) AddIgnoredService(serviceName, serviceNamespace string)
 	i.Services = append(i.Services, Service{Name: serviceName, Namespace: serviceNamespace})
 }
 
+// AddIgnoredApps add an app to the list of ignored apps.
+func (i *IgnoreWrapper) AddIgnoredApps(app ...string) {
+	i.Apps = append(i.Apps, app...)
+}
+
+// FieldSelector returns the field selectors query representing the ignored namespace and services.
+func (i *IgnoreWrapper) FieldSelector() string {
+	var selectors []string
+
+	for _, n := range i.Namespaces {
+		selectors = append(selectors, "metadata.namespace!="+n)
+	}
+
+	// We are loosing the filter by specifng namespace and service name here, but not sure if it's needed.
+	for _, n := range i.Services {
+		selectors = append(selectors, "metadata.name!="+n.Name)
```

Comment:
Not sure this is correct.

Services have a name/namespace combination, so we need to have some more logic in building this selector.

For example:
```
Ignore:
Namespace: kube-system
Service: default/kubernetes
```

Would result in:
`metadata.namespace!=kube-system, metadata.name!=kubernetes`
Which is close, but not right. 

I am not sure if field selectors are right for those complex filters

Reply:
Yeah, that's what I'm saying in the comment haha. I'll find a better way.

Reply:
Ended up removing field selectors and only relying on cache to build the initial validation.